src/styles/StyledCard.js

```diff
@@ -10,4 +10,7 @@ export const StyledCard = styled.div`
   box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25), 0 10px 10px rgba(0, 0, 0, 0.22);
   display: flex;
   flex-direction: column;
+  min-width: 142px;
+  max-width: 142px;
```

Comment:
Can we just replace these with `width: 142px`?

Reply:
Potentially. Kinda gets messy if you resize the screen too small but we can fix that with media rules

Reply:
```suggestion
  width: 142px;
```